### PR TITLE
Assignment1

### PR DESCRIPTION
**Pull request by NPThompson**

### CODE REVIEW

**Comment by cmartinezaraujo** [comment]
I will review this.

File: PrincessBrideTrivia/PrincessBrideTrivia.Tests/ProgramTests.cs [hunk 1]
```diff
@@ -6,6 +6,24 @@ namespace PrincessBrideTrivia.Tests
     [TestClass]
     public class ProgramTests
     {
+        [TestMethod]
```
Comment (cmartinezaraujo) [inline PrincessBrideTrivia/PrincessBrideTrivia.Tests/ProgramTests.cs:9]: Test method looks good. Nice work making use of the Generate QuestionsFile method.

File: PrincessBrideTrivia/PrincessBrideTrivia/Program.cs [hunk 2]
```diff
@@ -24,7 +24,7 @@ public static void Main(string[] args)
 
         public static string GetPercentCorrect(int numberCorrectAnswers, int numberOfQuestions)
         {
-            return (numberCorrectAnswers / numberOfQuestions * 100) + "%";
+            return (numberCorrectAnswers * 100) / numberOfQuestions + "%";
```
Comment (cmartinezaraujo) [inline PrincessBrideTrivia/PrincessBrideTrivia/Program.cs:27]: Nice test is passing now.

File: PrincessBrideTrivia/PrincessBrideTrivia/Program.cs [hunk 3]
```diff
@@ -89,6 +89,9 @@ public static Question[] LoadQuestions(string filePath)
                 question.Answers[1] = answer2;
                 question.Answers[2] = answer3;
                 question.CorrectAnswerIndex = correctAnswerIndex;
+
+                //write into return array
+                questions[i] = question;
```
Comment (cmartinezaraujo) [inline PrincessBrideTrivia/PrincessBrideTrivia/Program.cs:94]: Good this fixed the problem.

**Review by cmartinezaraujo — COMMENTED** [review]
Assignment
Pull Request title includes "Assignment 1" in the title ✔
Pull Request targets Assignment 1 branch ✔
Issue 1: Application no longer crashes ✔
Issue 2: Unit test properly passes ✔
Extra Credit
Issue 3: Feature request implemented ❌
Issue 3: Feature request unit tested ❌

Looks great nice work!

**Comment by breaman** [comment]
Nice work on the assignment, and unique way to get the GetPercentCorrect method working.

**Pull request closed without merging**